mkdocs.yml

```diff
@@ -1,8 +1,13 @@
 site_name: Freqtrade
 nav:
     - Home: index.md
-    - Installation Docker: docker.md
-    - Installation: installation.md
+    - Installation Docker: 
+        - Quickstart: docker.md
+        - Freqtrade with Docker Compose (Advanced): docker_compose.md
+        - Freqtrade without docker-compose: without_docker_compose.md
```

Comment:
I would see the compose variant as the simple one. 

Running docker without compose is quite complex, as you need to have the whole command identical for every command you run to not mess everything up.

Reply:
The `(Advanced)` contains only the advanced commands used in docker compose.

My current proposed structure is:

- Quickstart - a fast installation with docker composed (easy)
- Advanced compose - Compose, but for advanced users (optional)
- Docker without compose [hard]

Reply:
After having a look at the Quickstart / advanced compose sections, i think the 2 should really be combined.

They're not long enough to really justify seperation - and if you look at the docker-compose file we ship - EVERYONE will have to modify it at least in part as it hardcodes the `SampleStrategy`.